database/vertica/vertica_test.go

```diff
@@ -111,7 +111,7 @@ func TestOpen(t *testing.T) {
 			err bool
 		}{
 			{
-				addr + "?x-migrations-table=lolo",
+				addr + "&x-migrations-table=lolo",
```

Comment:
Modify `getVerticaURL()` to create the full url instead of building URLs by concatenating strings